<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Abrir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Guardar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QTableView" name="TablaAnalisis">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>80</y>
      <width>371</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView {
    border: 5px solid black;  /* Cambia el grosor y color del borde */
    border-radius: 10px;    /* Esquinas redondeadas */
    padding: 1px;           /* Espacio interno */
}</string>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="midLineWidth">
     <number>4</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Editor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>441</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid black;  /* Cambia el grosor y color del borde */
border-radius: 10px;    /* Esquinas redondeadas */
padding: 1px;           /* Espacio interno */</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Compilar">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Analizar</string>
    </property>
   </widget>
   <widget class="QLabel" name="CodigoAltoNivel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>CÃ“DIGO EN ALTO NIVEL</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="CodigoEnsamblador">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>441</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid black;  /* Cambia el grosor y color del borde */
border-radius: 10px;    /* Esquinas redondeadas */
padding: 1px;           /* Espacio interno */</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ensamblar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ensamblar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Enlazar">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>390</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Enlazar - Cargar</string>
    </property>
   </widget>
   <widget class="QTableView" name="SalidaEnlazarCargar">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>430</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView {
    border: 5px solid black;  /* Cambia el grosor y color del borde */
    border-radius: 10px;    /* Esquinas redondeadas */
    padding: 1px;           /* Espacio interno */
}</string>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="midLineWidth">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="RAM">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>RAM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Preprocesar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Preprocesar</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Preprocesado">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid black;  /* Cambia el grosor y color del borde */
border-radius: 10px;    /* Esquinas redondeadas */
padding: 1px;           /* Espacio interno */</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Abrir_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Guardar_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ResultadoEnsamblador">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>430</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid black;  /* Cambia el grosor y color del borde */
border-radius: 10px;    /* Esquinas redondeadas */
padding: 1px;           /* Espacio interno */</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1165</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
